### DISCUSSION TITLE
How can I select until the end of line and kill the meow way?

### DISCUSSION THREAD

**Discussion by arthsmn** [meow-edit/meow#617] [Q&A] [answered]
I'm new to meow but not to modal editing (vim and helix in the past). I was trying to figure out a way to mark something in multiple lines and then kill to the end of the line. An example would be removing comments like:
``` elisp
(dfndsoigfhs gerg) ; djhsdgfhghsidug
(djhfighbwr) ; difhniwoughpuiorg
(oijwsd feiojf)            ;djfhwiughviruewgfoie
```
Turning into this:
``` elisp
(dfndsoigfhs gerg)
(djhfighbwr)
(oijwsd feiojf)
```
I could achieve this selecting all the lines, grabbing, visiting `;`, entering insert-mode and then using `C-k`. But I would like to do it without entering insert-mode and with a meow key (not needing to press Ctrl). I tried after visiting use `L` to select and then `]l` to go to the end of the line, but it doesn't work, I've also tried using the simple `s` to kill everything after the cursor, but also got no luck. Is this something that can be done in meow easily or can be configured?

**Comment by underside** [comment, marked as answer]
Well, I see a couple of options here.

I Option (press 's')
1. Put cursor to the place after which you want to delete line
 (dfndsoigfhs gerg) ; <cursor here>   ;djhsdgfhghsidug
2. Press 's' - it will remove to the end of the line
3.  Repeat on each line

II Option (using Emacs 'query-replace-regexp')
1. Select text block with 'x' or any expand command  
1. Ctrl-Alt-5 (or M-x query-replase-regexp)
2. Use regexp, for your case ";.*"
3. Enter
4. confirm line change with 'y' (for each case) or '!' (replace in all places)

II option is much more usefull and flexible. 

P.S. Also you can try to use multiline editing in some cases, but in your particular case it's not convinient. 
To use multi line editing: Select text block -> 'G' -> edit -> 'G'

P.P.S Meow not like Evil, it not interfere (in most cases)  with Emacs keys, so I recommend to more active use Emacs options to editing.

**Reply by arthsmn** [reply]
Thanks for the answer, I'll integrate this native emacs command in my workflow!
